esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -226,23 +285,50 @@ void setup() {
   Serial.begin(115200);
   Serial.setDebugOutput(true);
 
-  FastLED.addLeds<LED_TYPE, DATA_PIN, COLOR_ORDER>(leds, NUM_LEDS);         // for WS2812 (Neopixel)
-  //FastLED.addLeds<LED_TYPE,DATA_PIN,CLK_PIN,COLOR_ORDER>(leds, NUM_LEDS); // for APA102 (Dotstar)
+  uint16_t milliAmps = (AVAILABLE_MILLI_AMPS < MAX_MILLI_AMPS) ? AVAILABLE_MILLI_AMPS : MAX_MILLI_AMPS;
+
+  #if PARALLEL_OUTPUT_CHANNELS == 1
```

Comment:
This looks somewhat ugly, but because FastLED requires these values to be template parameters, had to jump through some hoops.  See, for example, the definitions of template functions `LedOffset<>()` and `LedCount<>()`, which had to be `constexpr` compliant.

This was the simplest way I had to consolidate this aspect of the LED assignments.  I am open to alternatives.